packages/bindgen/src/templates/node-wrapper.ts

```diff
@@ -15,6 +15,7 @@
 // limitations under the License.
 //
 ////////////////////////////////////////////////////////////////////////////
```

Comment:
The changes in this file pertain to a bug with handling a difference between `nullptr` and the string `"null"`, as well as propagating errors thrown by the callback passed to the bindings `make_network_transport()`, into the promise chain.

Thanks to @RedBeard0531 for helping fix this interesting bug! 🐛 🚀